services/github/src/api.js

```diff
@@ -172,7 +172,7 @@ async function findTCChecks(github, owner, repo, branch, configuration) {
     }
     throw e;
   }
-  return checks.filter(checkObject => checkObject.app.id === configuration.github.credentials.appId);
+  return checks.filter(checkObject => checkObject.app.id === parseInt(configuration.github.credentials.appId));
```

Comment:
Maybe we can add radix to the parseInt function here `parseInt(x, 10)`

This is mostly not an issue in the latest ecmascript, but just to be safe

Reply:
Oh yeah, I wonder if it would take it as base 8 if the ID started with a 0, I'll fix that